### ISSUE TITLE
Error message on LDproxy_batch()

### ISSUE THREAD

**Issue by timyers** [CBIIT/LDlinkR#17]
Sent from user JM to NCI LDlink web admin email:

"I used LDproxy_batch to find LD proxies of a set of variants on May 14 and May 19. The May 14 run went well like this:
> LDproxy_batch(dat, pop = "CEU", r2d = "r2", token ='...', append = T)
 
LDlink server is working...
 
  error: rs1000470607 is not in 1000G reference panel.
 
LDlink server is working...
 
LDlink server is working...
 
That is, even though there were some variants that cannot be found in 1KG, they were simply skipped and the results for the rest of variants were produced correctly.
 
However, the May 19 run didn't go through:
> LDproxy_batch(dat, pop = "CEU", r2d = "r2", token ='...', append = T)
 
Submitting request for query variant rs10007826.
Checking status of server...
LDlink server is working...
 
Submitting request for query variant rs1001223416.
Checking status of server...
LDlink server is working...
 
Error in LDproxy(snp = snp[i, ], pop = pop, r2d = r2d, token = token,  :
    error: rs1001223416 is not in 1000G reference panel.
Calls: LDproxy_batch -> LDproxy
Execution halted
That is, once a variant that is not in 1KG was encountered, the function stopped and no results were produced for any variants."

**My note:**  possibly a result of the recent upgrade to the LDlinkR pkg that included support for genome build GRCh38.

**Comment by StephenJusto** [comment]
I am having the same issue 'error: rs555162510 Variant is not in 1000G reference panel'. Did you find a fix or an alternative?

**Comment by timyers** [comment]
There was an issue in spring of 2022 that halted execution if an error was encountered but was fixed.  But if you are using the most recent version, LDlinkR v1.2.3, you will be ok and `LDproxy_batch` should continue to execute.  Otherwise, the error output you experienced is expected since your query SNP, rs555162510, is not in the 1000G reference panel.  Below I discuss an option for handling that scenario.

If you would like to check whether any of your variants in a given set are not included in the 1000G reference panel, you might find the `ld_reflookup()` function from an R package available on [Github called `ieugwasr`](https://github.com/MRCIEU/ieugwasr) helpful.  This function interacts with the OpenGWAS API which houses LD reference panels for the 5 super-populations in the 1000G reference panel.  However, it only includes bi-allelic SNPs with a MAF > 0.01. The function will return an array of rsID's that are in the LD reference panel.  I included an example below where your rsID “rs555162510” is not in the reference but the "rs127" and "rs114" are.

> ieugwasr::ld_reflookup(c(rsid = "rs114", "rs555162510", "rs127"), pop = "EUR")
[1] "rs127" "rs114"
